employee-rostering-backend/src/main/java/org/optaweb/employeerostering/controller/TenantMockController.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright 2019 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.optaweb.employeerostering.controller;
+
+import java.util.Arrays;
+import java.util.List;
+
+import org.optaweb.employeerostering.domain.tenant.Tenant;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/rest/tenant")
+public class TenantMockController {
```

Comment:
Why not using a real tenant controller for integration testing? Is this a temporary solution before the controller is implemented?

Reply:
IIRC, the main reason for this class is so @cuijulian can see how the frontend handles the new backend. It is a temporary solution before the controller is implemented.

Reply:
Yes, as @Christopher-Chianelli has stated this is a temporary solution so we could test the new frontend with the new backend. The Tenant controller and service (ie. CRUD functionality) will be one of the last entities to be moved to Spring Boot, so this mock controller will be replaced by that once it's implemented.

I will be creating a PR for the entities I have moved to Spring Boot once Christopher merges his current PR. That PR will include the changes made in this one.